apt_offline_core/AptOfflineCoreLib.py

```diff
@@ -945,11 +946,16 @@ def fetcher( args ):
 
                 try:
                         for authstr in Str_HttpBasicAuth:
+                                # Add https schema if not provided any
+                                if not re.match('^[a-z]*:?//',authstr):
```

Comment:
You can remove this check and instead mandate that the user feed in the data in the correct form.